packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -545,6 +545,56 @@ const api = new apigateway.RestApi(this, 'books', {
   }
 })
 ```
+### Access Logging
+
+This is in addition to the detailed execution logs already provided by Amazon CloudWatch for API requests made to your APIs.
+The access logging feature lets you generate access logs in different formats such as CLF (Common Log Format), JSON, XML, and CSV.
+More info can be found [here](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html).
+
+Access logs can be configured to be sent to a specific CloudWatch log group. This can be configured either for all stages of the RestApi or for a specific stage.
+The following example will configure API Gateway to enable custom access logging.
+
+```ts
+// production stage
+const prdLogGroup = new cwlogs.LogGroup(this, "PrdLogs");
+const api = new apigateway.RestApi(this, 'books', {
+  deployOptions: {
+    accessLogDestination: new apigateway.CloudWatchLogsDestination(prdLogGroup),
```

Comment:
Update per the new name

Reply:
I'm sorry, I don't know what you want me to do.
I would like a few more comments.

Reply:
Replace `CloudWatchLogsDestination` with `LogGroupLogDestination`

Reply:
I understand!